manifest.yml

```diff
@@ -1,15 +1,26 @@
 ---
-applications:
-- name: registers-frontend
-  memory: 1GB
+defaults: &defaults
   buildpack: ruby_buildpack
-  instances: 2
-  domain: registers-trial.service.gov.uk
-  no-hostname: true
-  health-check-type: http
-  health-check-http-endpoint: /health_check/standard
+  memory: 1GB
   services:
     # postgres database
     - registers-frontend
     # environment variables (persisted for blue/green deploys)
     - registers-product-site-environment-variables
+
+applications:
+- name: registers-frontend
+  <<: *defaults
+  instances: 2
+  health-check-type: http
+  health-check-http-endpoint: /health_check/standard
+- name: registers-frontend-queue
+  <<: *defaults
+  instances: 1
+  command: bundle exec 'rake environment jobs:work'
+  health-check-type: process
+- name: registers-frontend-scheduler
+  <<: *defaults
+  instances: 1
+  command: bundle exec clockwork 'lib/clock.rb'
```

Comment:
Given that this is the instance downloading all the RSF, parsing it out and then updating the DB, it might need more than 1GB RAM, but this is something we can update later on if it's not appropriate at the moment.

Reply:
Actually I believe the `scheduler` just adds messages to the queue see https://github.com/Rykian/clockwork#use-with-queueing. It should need little memory and the `queue` instance should need more as it actually performs the job. It is easy to tweak this in future on a per-app basis.

Reply:
Sorry, this is what I meant, I put the comment on the wrong line. Sounds good.